ietf/templates/doc/shepherd_writeup.txt

```diff
@@ -95,9 +97,9 @@ to answer all of them.
 17. Are there any normative downward references (see [RFC 3967][10],
     [BCP 97][11])? If so, list them.
 
-18. Are there normative references to documents that are not ready for
-    advancement or are otherwise in an unclear state? If they exist, what is the
-    plan for their completion?
+18. Are there normative references to documents that are not ready to be
+    submitted to the IESG for publication or are otherwise in an unclear state?
+    If they exist, what is the plan for their completion?
```

Comment:
Is the sense "If they exist" missing a "do not" ?
Perhaps s/If they exist/If so/